rocAL/rocAL/include/meta_data/meta_node_resize.h

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright (c) 2019 - 2020 Advanced Micro Devices, Inc. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#pragma once
+#include <set>
+#include <memory>
+#include "bounding_box_graph.h"
+#include "meta_data.h"
+#include "node.h"
+#include "node_resize.h"
+#include "parameter_vx.h"
+class ResizeMetaNode:public MetaNode
+{
+    public:
+        ResizeMetaNode() {};
+        void update_parameters(pMetaDataBatch input_meta_data, pMetaDataBatch output_meta_data) override;
+        std::shared_ptr<ResizeNode> _node = nullptr;
+    private:
+        float _dst_to_src_width_ratio, _dst_to_src_height_ratio;
```

Comment:
Why there is no initialize function as in TOT?

Reply:
Why are these 2 private to the class? Can it not be local?

Reply:
Initialize function is used for resizing vectors to the batch size. There are no vectors used in the metanode resize since we are just rescaling the metadata here

Reply:
So it is ok to ignore initialize() API even though it is empty?

Reply:
Has this been addressed.

Reply:
I added a initialize() API in meta_node_resize.h with empty definition